container/openejb-loader/src/main/java/org/apache/openejb/observer/event/ObserverRemoved.java

```diff
@@ -26,10 +26,20 @@
 
     private final Object observer;
 
+    /**
+     * Remove the observable object so this object doesn't have any observers
```

Comment:
I don't think that's true. It is constructor, and it apparently does not remove anything.

Reply:
This is actually the event thrown when an observer is removed. All remaining observers have the ability to observe this event and react if needed.